### PR TITLE
Wrangler v2

### PR DESCRIPTION
**Pull request by KevinJoiner**
## Issue: https://github.com/rancher/rancher/issues/43618

This PR set the version of Wrangler used to be a Wrangler v2 tag.
This PR also updates all of the wrangler import paths to include `/v2` prefix

These changes are needed so Rancher/Rancher can move to a tagged version for Wrangler. Since Rancher imports and shares wrangler schemas with channelserver, they must be wrangler/v2 schemas.
https://github.com/rancher/rancher/blob/1ba6347a474547ff683d9c09a073e6bf675faa7f/pkg/channelserver/channelserver.go#L95C1-L95C1

I am unsure how this would affect other projects using an untagged version of Wrangler v2.

# Changes made between wrangler version.
https://github.com/rancher/wrangler/compare/v1.0.2...v2.0.2

### CODE REVIEW

File: go.mod
Comment (KevinJoiner) [inline go.mod]: Remove and update apiserver commit after https://github.com/rancher/apiserver/pull/47 is merged

**Review by MbolotSuse — COMMENTED** [review]
LGTM. Can approve after the replace directive is removed. 

Would also recommend getting another reviewer on this who has contributed to this repo before.

**Review by MbolotSuse — APPROVED** [review]
LGTM now that the fork has been replaced.

**Review by Oats87 — APPROVED** [review]

**Comment by chiukapoor** [comment]
@Oats87 is this PR ready to be merged?

`channel-server` with wrangler/v2 changes is required for k8s 1.28 support https://github.com/rancher/rancher/issues/43109

**Comment by Oats87** [comment]
> @Oats87 is this PR ready to be merged?
> 
> `channel-server` with wrangler/v2 changes is required for k8s 1.28 support [rancher/rancher#43109](https://github.com/rancher/rancher/issues/43109)

I just merged it

**Pull request merged**
